app/components/post_draft/send_handler/send_handler.test.tsx

```diff
@@ -193,42 +194,4 @@ describe('components/post_draft/send_handler/SendHandler', () => {
             }));
         });
     });
-
-    it('should execute sendMessageHandler when send_draft_button is clicked', async () => {
```

Comment:
Why do we need to remove this test?

Reply:
This was testing the same thing as in the above test.